### PR TITLE
Event feedback form

### PR DESCRIPTION
**Pull request by willwull**
This PR adds a form to generate event feedback in the route `/create-event-feedback`, and the result can be viewed in `/event-feedback`.

Since I had to implement new buttons and inputs, I updated the login screen as well.

(The code is garbage sorry 😵)

### CODE REVIEW

File: app/containers/CreateEventFeedback/template.js [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+    type: 'response',
+  },
+]
+
+function scaleToYesNo(scaleData: Array<any> = []) {
+  let yes = 0
+  let no = 0
+  scaleData.forEach((amount, index) => {
+    if (index + 1 >= 3) {
+      yes += Number.parseInt(amount, 10)
```
Comment (mictab) [inline app/containers/CreateEventFeedback/template.js]: Don't think we need the `Number` prefix?

File: app/containers/CreateEventFeedback/template.js [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    title: 'What did you enjoy the most about the event and the company?',
+    type: 'response',
+  },
+  {
+    title: 'What could have been improved?',
+    type: 'response',
+  },
+]
+
+function scaleToYesNo(scaleData: Array<any> = []) {
```
Comment (mictab) [inline app/containers/CreateEventFeedback/template.js]: What does this scaleData array contain? Any? ;)

File: app/containers/CreateEventFeedback/template.js [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+}
+
+/**
+ * Takes form data from the event feedback form and inserts
+ * them into the template.
+ *
+ * @param {Object} formData
+ */
+export function addResponses(formData: Object) {
+  console.log(formData)
```
Comment (mictab) [inline app/containers/CreateEventFeedback/template.js]: Remove this.

File: app/containers/CreateEventFeedback/template.js [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+  const data: Array<Object> = [...template].map(question => {
+    console.log(question)
```
Comment (mictab) [inline app/containers/CreateEventFeedback/template.js]: This as well.

File: app/containers/CreateEventFeedback/template.js [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+          .filter(txt => txt.length > 0), // filter out empty lines
+      }
+    }
+
+    return {
+      ...question,
+      datasets: [
+        {
+          data: formData[question.title],
+          backgroundColor,
```
Comment (mictab) [inline app/containers/CreateEventFeedback/template.js]: What's this for?

File: app/containers/CreateEventFeedback/template.js (hunk 1 above)
Comment (willwull) [reply app/containers/CreateEventFeedback/template.js]: Ah you're right, the ESLint config in some previous project I've worked on needed it, but ours doesn't. Will clean up.

Comment (mictab) [reply app/containers/CreateEventFeedback/template.js]: You can add that to the lint if you think it's a good change.

**Pull request merged**